src/System.ValueTuple/src/System/Numerics/Hashing/Hash.cs

```diff
@@ -0,0 +1,101 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace System.Numerics.Hashing
+{
+    internal static class Hash
```

Comment:
Why is this class valuable here? Why not just delete this and revert all of the changes to ValueTuple, other than using the new HashHelpers.Combine?  I'm not seeing what the value is (no pun intended :))